api/formdata.go

```diff
@@ -101,12 +97,21 @@ func (f *FormDataRequest) validate(attachmentValidator FormDataValidator) (errs
 				errs = append(errs, validatorError(FileFieldKey, msg))
 			}
 
-			if data, err = ioutil.ReadAll(file); err != nil {
+			tmpZip, err := os.CreateTemp("", "s3-zip_*.zip")
```

Comment:
must remove it after use :)

Reply:
pls check but i think this is ok? https://github.com/ONSdigital/dp-interactives-api/blob/87d6cb6d056fb1745e1d0458164324073b1e67d8/api/interactives.go#L44 - so end of request